README.md

```diff
@@ -386,6 +386,14 @@ Skip the fetching of blocklists from any URLs that are configured.
 
 Skip the fetching of blocklists from any remote instances that are configured.
 
+### override_private_comment
+
+Defaults to None.
+
+Stamp all *new* blocks pushed to a remote server with this comment or code. 
+Helps to identify blocks you've created on a server via this process versus ones that
```

Comment:
I'm a little unclear on what `this process` refers to here.

Does this mean "when you use FediBlockHole to import blocklists"?